### ISSUE TITLE
Responsive Design Mode toggle back to default mode not working consistantly

### ISSUE THREAD

**Issue by gillmourtunhira** [zen-browser/desktop#2268]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

When you toggle desktop to mobile view or in responsive design mode (ctrl+shift+m), you cannot click to toggle back from responsive mode whether clicking the icon or using shortcut at some instances,

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.12

### What platform are you seeing the problem on?

Linux

**Comment by ninoDeme** [comment]
i think i have been experiencing this problem, the error seems to be caused by the tab unloader unloading tabs while they are in Responsive Design Mode.

**Comment by salamijason** [comment]
hey are you still getting this issue? I use the responsive design mode quite a bit myself and haven't gotten the issue before? if you're still getting it, could you send a screen recording?

**Comment by ninoDeme** [comment]
> hey are you still getting this issue? I use the responsive design mode quite a bit myself and haven't gotten the issue before? if you're still getting it, could you send a screen recording?

[Screencast From 2024-11-01 13-40-05.webm](https://github.com/user-attachments/assets/278a55a5-ced4-4ab4-8f75-f4677edda83a)

![image](https://github.com/user-attachments/assets/14432545-ac5e-42bd-a8de-de8840277f89)

**Comment by Aspireve** [comment]
Heyy Hi @salamijason!!

While using the application for development, I too faced a similar issue with the toggle of the responsive layout. Btw I would like to try to resolve it too

PS: This is on a windows based system

https://github.com/user-attachments/assets/fba5a1d6-b3e5-465d-a80e-bbb681c22a3e

**Comment by gillmourtunhira** [comment]
@salamijason yep still experiencing it, as shown by @Aspireve

**Comment by salamijason** [comment]
Unable to replicate on windows. will ask one of the handlers on linux to verify. thanks for the info!

**Comment by RayZ3R0** [comment]
https://github.com/user-attachments/assets/099d33ed-2730-45b8-b4e5-69fa8dec2d0e

Weird, its not happening to me either, on linux

**Comment by Aspireve** [comment]
Yes I guess its really difficult to recreate, I guess it happens only in some scenarios

**Comment by salamijason** [comment]
Are the two of you using any other config settings that could be causing this? Could you guys check if this happens with every site?

**Comment by ninoDeme** [comment]
i think i have been experiencing this problem, the error seems to be caused by the tab unloader unloading tabs while they are in Responsive Design Mode.

[Screencast From 2024-11-01 13-40-05.webm](https://github.com/user-attachments/assets/278a55a5-ced4-4ab4-8f75-f4677edda83a)

![image](https://github.com/user-attachments/assets/14432545-ac5e-42bd-a8de-de8840277f89)

**Comment by salamijason** [comment]
Yes. We couldn't replicate the issue at all on our side. It'd be great to figure out in which specific scenarios this bug occurs.

**Comment by LordzShadow** [comment]
Hi! I'm having the same issue.

1. Open the responsive design mode for a tab
2. Go to another tab
3. Unload the previous tab (either manually or after a while)
4. Go back to the unloaded tab
5. Responsive design mode is stuck and can't be disabled without  closing the tab and reopening it

I'm running the browser on Manjaro KDE 24.2.0, using the AppImage version of Zen. Haven't edited any configs.
It's quite an edge case and not a major issue, but would be cool to see it fixed as I often forget to disable it :grinning: 

If you would like to see any logs, I'd be happy to provide them, I just don't know where to look. :)

![image](https://github.com/user-attachments/assets/4a4d2a15-f847-40ad-80f8-ab16ad5c66ec)

Edit: The same issue persists on the AUR build

**Comment by salamijason** [comment]
I see. Thanks for the info! will send to big dev cheff

**Comment by mr-cheffy** [comment]
You do you even encounter an issue like this...

**Comment by salamijason** [comment]
Thread is a bit dead but before i close it, has there been any updates?

**Comment by gillmourtunhira** [comment]
> Thread is a bit dead but before i close it, has there been any updates?

still waiting for your feedback if the issue was fixed by the devs, as on Dec 10, 2024 you highlighted you are moving it to "big dev cheff"

**Comment by salamijason** [comment]
Right, thing is i don't believe it is actively worked on right now and so I wanted to check if the bug still occurs in the meantime.

**Comment by LordzShadow** [comment]
Still present in 1.7.2b (Firefox 134.0.2) (64-bit)

**Comment by tostesdaniel** [comment]
Just downloaded the browser yesterday and can confirm it still happens on Windows latest version

**Comment by LordzShadow** [comment]
Still relevant, could reproduce on MacOS 15.3.1 with Zen 1.9b (Firefox 136.0) (aarch64)

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Responsive%20Design%20Mode%20toggle%20back%20to%20default%20mode%20not%20working%20consistantly%0A
